src/App.tsx

```diff
@@ -1,14 +1,51 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Filter } from './components/Filter/Filter';
+import { Notification } from './components/Notification';
+import { TodoList } from './components/TodoList';
+import { ErrorMessage } from './types/ErrorMessage';
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState(ErrorMessage.None);
+  const [status, setStatus] = useState(Status.All);
+
+  let visibleTodos = [...todos];
+
+  if (status !== Status.All) {
+    visibleTodos = visibleTodos.filter(todo => {
+      switch (status) {
+        case Status.Active:
+          return !todo.completed;
+        case Status.Completed:
+          return todo.completed;
+        default:
```

Comment:
if you have typization you won't get invalid status, so you can move `All` case to default